src/tests/ftest/scrubber/frequency.py

```diff
@@ -0,0 +1,82 @@
+            Test scrubber frequency can be dynamically updated.
+
+            1. Create a test pool without scrubber properties.
+               The default scrub frequency is set to 1 week.
+            2. Initial scrub happens when the objects are created.
+            3. Check objects are not scrubbed after initial scan.
+               Wait for 5 minutes and see scrubber not doing
+               any more object scrubbing now.
+            4. Update the scrubber frequency to 5 seconds.
+            5. Observe the objects are scrubbed every 5 seconds.
+
+        :avocado: tags=all,pr,daily_regression
+        :avocado: tags=hw,medium
+        :avocado: tags=scrubber
+        :avocado: tags=TestWithScrubberFreq,test_objects_scrubbed_properly
+        """
+        flags = self.params.get("ior_flags", '/run/ior/iorflags/*')
+        apis = self.params.get("ior_api", '/run/ior/iorflags/*')
+        transfer_block_size = self.params.get("transfer_block_size",
+                                              '/run/ior/iorflags/*')
+        obj_class = self.params.get("obj_class", '/run/ior/iorflags/*')
+        pool_prop = self.params.get("properties", '/run/pool/*')
+        cont_prop = self.params.get("properties", '/run/container/*')
+        self.ior_cmd.api.update(apis[0])
+        self.ior_cmd.flags.update(flags[0], "ior.flags")
+        self.ior_cmd.dfs_oclass.update(obj_class[0])
+        self.ior_cmd.dfs_dir_oclass.update(obj_class[0])
+        self.create_pool_cont_with_scrubber(pool_prop=pool_prop, cont_prop=cont_prop)
+        for test in transfer_block_size:
+            self.ior_cmd.transfer_size.update(test[0])
+            self.ior_cmd.block_size.update(test[1])
+        # Run IOR and gather the total scrubbed metrics information.
+        self.run_ior_and_check_scruber_status(pool=self.pool, cont=self.container)
+        # Wait for 5 minutes to get first scrubber bytes scrubbed metrics.
+        # NOTE: This value could change depending on the IOR data (objects created)
+        self.log.info("Sleeping for 5 minutes pool property set to scrub:timed")
+        sleep(300)
+        self.log.info("Initial scrubber metrics with scrub:timed")
+        initial_scrubbed_metrics = self.scrubber.get_scrubber_bytes_scrubbed_total()
+        self.log.info("Sleeping another 5 minutes pool property set to scrub:timed")
+        sleep(300)
+        self.log.info("Final scrubber metrics with scrub:timed")
+        final_scrubbed_metrics = self.scrubber.get_scrubber_bytes_scrubbed_total()
+        status = self.verify_scrubber_metrics_value(initial_scrubbed_metrics,
+                                                    final_scrubbed_metrics)
+        if status is True:
+            self.fail("--Test Failed: Metrics Value is Changing--")
+        # Now set the scrub-freq to 5 seconds.
+        self.pool.set_property("scrub-freq", "5")
+        self.log.info("Sleeping for 5 secs pool property scrub:timed,scrub-freq:5")
+        sleep(5)
```

Comment:
Why do we sleep 5 sec here? If the scrubbing happens during this 5 sec, wouldn't `initial_scrubbed_metrics` and `final_scrubbed_metrics` become equal? What if we obtain `initial_scrubbed_metrics` before setting the frequency to 5 sec?